airflow/providers/amazon/provider.yaml

```diff
@@ -516,6 +516,9 @@ hooks:
       - airflow.providers.amazon.aws.hooks.appflow
 
 triggers:
+  - integration-name: Amazon Web Services
+    python-modules:
+      - airflow.providers.amazon.aws.triggers.base_trigger
```

Comment:
Do we need the `_trigger` suffix?

For example we have:
`from airflow.hooks.base import BaseHook`

Reply:
hmm, yeah, I could name the file `base` instead of `base_trigger`, but what would it change ? How is it better ?

Reply:
When it comes to naming things my philosophy is the shorter - the better (as long as not compromising name being self explained)